### DISCUSSION TITLE
Folder Creation and Permissions

### DISCUSSION THREAD

**Discussion by d0nizam** [coobbi/Jre4android#5] [General]
I am experiencing an issue with the JRE4Android app (version 1.6.30) on my Samsung Galaxy A54 5G running Android 14. When I attempt to run a specific .jar file, the app is supposed to create folders to facilitate its operation, but it does not create these folders. 

Additionally, the app does not prompt me to grant writing permissions.Here are the details:App Version: 1.6.30Device Model: Samsung Galaxy A54 5GAndroid Version: 14

Issue Description: The app fails to create the required folders and does not request writing permissions when attempting to run a specific .jar file.I have reviewed the app’s permissions on my device, and it appears that writing permissions are not being requested. 

Could you please provide guidance on how to resolve this issue or if there is a workaround available?Thank you for your assistance.

**Comment by coobbi** [comment]
@DzhaidNizam 
https://github.com/coobbi/Jre4android/releases/download/pre_free_release/app_launcher-free-release.apk

Try this, there are permissions setting at app setting page which contains writing permission
